master/usage/ipv6.md

```diff
@@ -1,20 +1,155 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
+{{site.prodname}} supports connectivity over IPv6, for hosts and
```

Comment:
https://docs.projectcalico.org/v3.0/usage/decommissioning-a-node

I'd like to start aligning the structure of our usage documentation. ^ I think the decommissioning a node document is a good reference point to start with.

One important thing is that each document starts by concisely describing exactly why one might want to follow this document, so users can tell really quickly whether or not it is worthwhile for them to read the rest of the document.